test/libc/stdio/fscanf_test.c

```diff
@@ -0,0 +1,33 @@
+/*-*- mode:c;indent-tabs-mode:nil;c-basic-offset:2;tab-width:8;coding:utf-8 -*-│
+│ vi: set et ft=c ts=2 sts=2 sw=2 fenc=utf-8                               :vi │
+╞══════════════════════════════════════════════════════════════════════════════╡
+│ Copyright 2022 Justine Alexandra Roberts Tunney                              │
```

Comment:
You can own the copyright for your own test files, feel free to set the year and your own name when creating a new test file.

Reply:
Thanks, I just pushed a fix! I thought of that when I was making this PR but decided it would be weird to claim the copyright for, like, 10 lines of code.

Reply:
Not weird at all. It leaves a record that you were here and that you helped.

You can also own non-test code if you're contributing a feature. For example, let's say you discovered a faster better way to implement `log10()`. You can rewrite `log10.c` from scratch with your discovery, put your name on the top, and add a `__notice()` directive so that your name will even be embedded inside every binary that uses `log10()`. Stuff like this is obviously an exception to the rule, so just ask me beforehand if you want to do it.